config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-main.yaml

```
@@ -22,6 +22,32 @@ periodics:
     testgrid-tab-name: capi-test-main
     testgrid-alert-email: sig-cluster-lifecycle-cluster-api-alerts@kubernetes.io
     testgrid-num-failures-to-alert: "4"
+- name: periodic-cluster-api-test-mink8s-main
+  interval: 1h
+  decorate: true
+  labels:
+    preset-service-account: "true"
+  extra_refs:
+  - org: kubernetes-sigs
+    repo: cluster-api
+    base_ref: main
+    path_alias: sigs.k8s.io/cluster-api
+  spec:
+    containers:
+    - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20211217-ea95cec1d4-1.23
```

```suggestion
    - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220221-c13e827224-1.23
```
Don't ask me how, but apparently I picked up the version I recommended from some outdated branch

Please also directly squash after this change, PR should be ready to merge then.